routes/error_codes.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module ErrorCodes
```

Comment:
https://github.com/ubicloud/ubicloud/pull/162#discussion_r1255227526

We might start to embed http error code, message etc. to exception class